about.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./cap.css" />
+    <link rel="stylesheet" href="style.css">
+    <script
+      src="https://kit.fontawesome.com/38b6d4ab77.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>BOOK FESTIVAL</title>
+</head>
+<body>
+    <header>
+        <nav class="top-nav">
+          <ul class="nav-list">
+            <li>
+              <a href="#" class="nav-item"><i class="fa fa-facebook"></i></a>
+            </li>
+            <li>
+              <a href="#" class="nav-item"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li><a href="#" class="nav-item">English</a></li>
+            <li><a href="#" class="nav-item">My Page</a></li>
+            <li><a href="#" class="nav-item">Logout</a></li>
+          </ul>
+        </nav>
+  
+        <nav class="main-nav">
+          <a href="./index.html" class="logo"
+            ><img src="./assets/bookIcon-removebg-preview.png" alt="ICPC-Logo"
+          /></a>
+          <ul class="main-nav-list nav-list">
+            <li><a href="#" class="main-nav-item nav-item" id="close-menu">X</a></li>
+            <li><a href="./index.html" class="main-nav-item nav-item">Home</a></li>
+            <li><a href="#logo-section" class="main-nav-item nav-item">Logo Detail</a></li>
+            <li><a href="#past-section" class="main-nav-item nav-item">Past Events</a></li>
+            <li><a href="#partner" class="main-nav-item nav-item">Sponser</a></li>
+            <li><a href="#" class="main-nav-item nav-item">News</a></li>
+            <li><a href="#" class="nav-item nav-btn">Campaign</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+        <section class="intro-section">
+            <div class="mobile-nav-head">
+                <a href="#" class="menuIcon"><i class="fa-solid fa-bars"></i></a>
+                
+                <div class="mobile-nav-head logo-title">
+                <a href="/index.html" class="logo-head"><img src="/assets/bookIcon-removebg-preview.png" alt=""></a>
+                <h3>BOOK FEST</h3>
+                </div>
+              </div>
+           
+            <div class="container">
+               
+                <div class="about">
+                    <h2 class="intro-preheading">"Are you ready for this Opportunity?"</h2>
+                    <h1 class="main-heading">
+                        Official BOOK FESTIVAL
+                    </h1>
+                    <p class="intro">The 2023 Call for nominations TEXCHANGE Melina Mercouri 

```

Comment:
- [ ] Just as the previous reviewer noted, I also noticed you have two different text, whereas the design template has one but on two lines, kindly replicate this in yours too.

### Your project
![image](https://github.com/Adesina-Abiodun/Capstone-project/assets/86200356/f4cac54c-e529-43cc-856d-c4cee702c1f6)

### Design template
![image](https://github.com/Adesina-Abiodun/Capstone-project/assets/86200356/4b165d73-d4b5-425f-a062-ff6dc846126c)